### DISCUSSION TITLE
[Windows] remote tunneld for iOS17.3

### DISCUSSION THREAD

**Discussion by sk1kgn** [doronz88/pymobiledevice3#981] [Q&A] [answered]
version: 4.5.2 platform: Windows
after use cammand "pymobiledevice3 remote tunneld", when iphone connect to the PC, it will create tunnel immediately(so cool!),
but this requires the phone OS-version >= 17.4，if iOS < 17.4， it still need WeTest driver, and create tunnel will takes a lot of time.

My question is, Will this be optimized for iOS < 17.4 in future?
thanks 👍 : )

**Comment by doronz88** [comment, marked as answer]
Unless someone comes forth with a full open-source driver then the answer is no. This is not something in the project's scope.

**Reply by sk1kgn** [reply]
thanks for your reply, it's helpful for my update plan.
